src/index.js

```diff
@@ -0,0 +1,253 @@
+    <label class="form-check-label ml-4 position-relative" for="task${index}">
+    ${name}
+    <button data-toggle="modal" data-target="#editTask" class='btn expand-btn btn-link p-0 d-none'><i class="fas fa-pen"></i></button>
+    </label>
+    `;
+  } else {
+    liElemt.innerHTML = `
+    <input class="form-check-input ml-1" checked type="checkbox" data="${index}" value="" id="task${index}">
+    <label class="form-check-label done ml-4 position-relative" for="task${index}">
+    ${name}
+    <button data-toggle="modal" data-target="#editTask" class='btn expand-btn btn-link p-0 d-none'><i class="fas fa-pen"></i></button>
+    </label>
+    <button class='btn btn-link trash p-0 d-inline'><i class="far fa-trash-alt "></i></button>
+    `;
+    liElemt.children[2].addEventListener('click', function removeATask() {
+      mainProjects.projectsArr[projectId].removeTask(
+        parseInt(this.previousElementSibling.previousElementSibling.getAttribute('data'), 10) - 1,
+      );
+      this.parentElement.classList.add('animate-out');
+      renderProjects(mainProjects.projectsArr);
+      setTimeout(() => {
+        renderTask(
+          mainProjects.projectsArr[projectId].tasksArr,
+          mainProjects.projectsArr[projectId].taskName,
+        );
+      }, 200);
+      localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+    });
+  }
+  liElemt.children[0].addEventListener('click', function toggleCompleteOnATask() {
+    mainProjects.projectsArr[projectId].toggleComplete(parseInt(this.getAttribute('data'), 10) - 1);
+    renderProjects(mainProjects.projectsArr);
+    renderTask(
+      mainProjects.projectsArr[projectId].tasksArr,
+      mainProjects.projectsArr[projectId].taskName,
+    );
+    localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+  });
+  liElemt.addEventListener('mouseenter', () => {
+    const editBtn = liElemt.querySelector("[data-toggle='modal']");
+    editBtn.classList.add('d-inline');
+    editBtn.classList.remove('d-none');
+  });
+
+  liElemt.addEventListener('mouseleave', () => {
+    const editBtn = liElemt.querySelector("[data-toggle='modal']");
+    editBtn.classList.add('d-none');
+    editBtn.classList.remove('d-inline');
+  });
+  liElemt.querySelector('.expand-btn').addEventListener('click', function expandBtn() {
+    const indexOfTask = parseInt(this.parentElement.previousElementSibling.getAttribute('data') - 1, 10);
+    const { name } = mainProjects.projectsArr[projectId].tasksArr[indexOfTask];
+    const { description } = mainProjects.projectsArr[projectId].tasksArr[indexOfTask];
+    // let priority = mainProjects.projectsArr[projectId].tasksArr[indexOfTask].priority;
+    const { date } = mainProjects.projectsArr[projectId].tasksArr[indexOfTask];
+
+    fillEditTaskForm(name, description, 'low', date);
+    addEventListenerToEditTaskForm(indexOfTask);
+  });
+  return liElemt;
+}
+
+// testing 321
```

Comment:
- [ ] This is not a commented code but it's not an informative comment. We should also delete this.